### DISCUSSION TITLE
json settings stored in the image note

### DISCUSSION THREAD

**Discussion by 42degrees** [Comfy-Org/ComfyUI#1641] [Ideas]
I looked through the nodes that I could add and didn't find any related to image meta-data.  It seems like it would be great to be able to have the images store the same information you can export as a json file right in a meta-data chunk.  Of course, then I would want to be able to take an image and load the json from the meta-data chunk.  Does this already exist?  

I am a programmer, how hard is it to write a node that had access to the json exporter and the PNG file after it has been generated?

**Comment by 42degrees** [comment]
OK, I guess I should have tried it.  I did look in the generated PNG images and didn't see any meta-data, but if I drop one on a browser window it appears to load the original settings.  Not sure how that works, but it does.  Thanks guys for a very cool tool!

**Reply by phineas-pta** [reply]
it's literally written in README 😅

**Comment by 42degrees** [comment]
> it's literally written in README 😅

Damn, it is right there in the notes.  I totally missed it.  

Also, I've seem images of a configuration that can restore the displayed configuration.  I haven't seen how to do that yet either, is that equally obvious and I'm missing it?
